### ISSUE TITLE
TencentCloudSDKError  Code=OperationDenied.InstanceOperationInProgress

### ISSUE THREAD

**Issue by virain** [hashicorp/packer#9054]
#### Overview of the Issue

```
==> tencentcloud-cvm: Trying to create a new image: redis...
==> tencentcloud-cvm: Failed to create image: [TencentCloudSDKError] Code=OperationDenied.InstanceOperationInProgress, Message=The instance `['ins-meu2jpya']` already has operation in progress. Please try again later., RequestId=1996707b-c2f3-4213-b97d-ebe46ff5dcf4
==> tencentcloud-cvm: Provisioning step had errors: Running the cleanup provisioner, if present...
==> tencentcloud-cvm: Deleting instance because of error...
==> tencentcloud-cvm: Failed to terminate instance(ins-meu2jpya), please delete it manually: [TencentCloudSDKError] Code=OperationDenied.InstanceOperationInProgress, Message=The instance `['ins-meu2jpya']` already has operation in progress. Please try again later., RequestId=fe4ed33d-40d8-4c0b-945d-58708b920594
==> tencentcloud-cvm: Deleting securitygroup because of error...
2020/04/13 16:44:12 packer-builder-tencentcloud-cvm plugin: 2020/04/13 16:44:12 retry.go:66: Retryable error: [TencentCloudSDKError] Code=ResourceInUse, Message=The specified resource `sg-7y3fi8e3` is already in use., RequestId=3e5b947c-8ca5-408c-ac8c-aa2ad36175f6
2020/04/13 16:44:14 packer-builder-tencentcloud-cvm plugin: 2020/04/13 16:44:14 retry.go:66: Retryable error: [TencentCloudSDKError] Code=ResourceInUse, Message=The specified resource `sg-7y3fi8e3` is already in use., RequestId=0bc36805-f52a-49c3-a655-3b9634b49a34
2020/04/13 16:44:16 packer-builder-tencentcloud-cvm plugin: 2020/04/13 16:44:16 retry.go:66: Retryable error: [TencentCloudSDKError] Code=ResourceInUse, Message=The specified resource `sg-7y3fi8e3` is already in use., RequestId=7f5e873d-b6b7-4d1c-98f8-717fa55a8c5b
```

### Packer version

`1.5.5`

### Packer Buildfile
```
{
  "builders": [
    {
      "type": "tencentcloud-cvm",
      "secret_id": "*****,
      "secret_key": "*****",
      "region": "ap-hongkong",
      "zone": "ap-hongkong-1",
      "instance_type": "S2.MEDIUM4",
      "source_image_id": "img-9qabwvbn",
      "ssh_username": "root",
      "image_name":"redis",
      "disk_type": "CLOUD_SSD",
      "force_poweroff": true,
      "packer_debug": true,
      "associate_public_ip_address": true
    }
  ],
  "provisioners": [
    {
      "type": "shell",
      "inline": [
        "sleep 30",
        "yum install redis.x86_64 -y"
      ]
    }
  ]
}

```

### Operating system and Environment details

macOS 10.15.4 

### Log Fragments and crash.log files

```
==> tencentcloud-cvm: Trying to detach keypair: skey-nipiw0ov...
    tencentcloud-cvm: Keypair detached
==> tencentcloud-cvm: Trying to create a new image: redis...
==> tencentcloud-cvm: Failed to create image: [TencentCloudSDKError] Code=OperationDenied.InstanceOperationInProgress, Message=The instance `['ins-meu2jpya']` already has operation in progress. Please try again later., RequestId=1996707b-c2f3-4213-b97d-ebe46ff5dcf4
==> tencentcloud-cvm: Provisioning step had errors: Running the cleanup provisioner, if present...
==> tencentcloud-cvm: Deleting instance because of error...
==> tencentcloud-cvm: Failed to terminate instance(ins-meu2jpya), please delete it manually: [TencentCloudSDKError] Code=OperationDenied.InstanceOperationInProgress, Message=The instance `['ins-meu2jpya']` already has operation in progress. Please try again later., RequestId=fe4ed33d-40d8-4c0b-945d-58708b920594
==> tencentcloud-cvm: Deleting securitygroup because of error...
2020/04/13 16:44:12 packer-builder-tencentcloud-cvm plugin: 2020/04/13 16:44:12 retry.go:66: Retryable error: [TencentCloudSDKError] Code=ResourceInUse, Message=The specified resource `sg-7y3fi8e3` is already in use., RequestId=3e5b947c-8ca5-408c-ac8c-aa2ad36175f6
2020/04/13 16:44:14 packer-builder-tencentcloud-cvm plugin: 2020/04/13 16:44:14 retry.go:66: Retryable error: [TencentCloudSDKError] Code=ResourceInUse, Message=The specified resource `sg-7y3fi8e3` is already in use., RequestId=0bc36805-f52a-49c3-a655-3b9634b49a34
2020/04/13 16:44:16 packer-builder-tencentcloud-cvm plugin: 2020/04/13 16:44:16 retry.go:66: Retryable error: [TencentCloudSDKError] Code=ResourceInUse, Message=The specified resource `sg-7y3fi8e3` is already in use., RequestId=7f5e873d-b6b7-4d1c-98f8-717fa55a8c5b
2020/04/13 16:44:21 packer-builder-tencentcloud-cvm plugin: 2020/04/13 16:44:21 retry.go:66: Retryable error: [TencentCloudSDKError] Code=ResourceInUse, Message=The specified resource `sg-7y3fi8e3` is already in use., RequestId=aa719a79-d540-47cc-8bf9-0778dfcfeb98
2020/04/13 16:44:24 packer-builder-tencentcloud-cvm plugin: 2020/04/13 16:44:24 server.go:85: Received interrupt signal (count: 1). Ignoring.
```

**Comment by virain** [comment]
@likexian Help

**Comment by likexian** [comment]
Hello @virain 
Thanks for your feedback, I will do on it.

**Comment by sylviamoss** [comment]
Hello @virain, @likexian  has opened a PR to fix this. Could you test the binaries and let us know if the changes work for you? Here the binaries https://circleci.com/gh/hashicorp/packer/45661#artifacts/containers/0

**Comment by virain** [comment]
@sylviamoss  is work , Thanks  :+1:

**Comment by virain** [comment]
Hello @sylviamoss is work! 👍

**Comment by nywilken** [comment]
Closed by #9069 thanks @virain @sylviamoss
